src/lib/block_producer/block_producer.ml

```diff
@@ -606,17 +608,15 @@ module Vrf_evaluation_state = struct
     | Completed ->
         t.vrf_evaluator_status <- Completed
 
-  let poll ~vrf_evaluator ~logger t =
+  let poll ~vrf_evaluator ~logger ~vrf_poll_interval_ms t =
```

Comment:
Consider changing parameter to `~(vrf_poll_interval : Time.Span.t)` (and performing conversion somewhere on top level)